internal/tiltfile/skylarkTypes.go

```diff
@@ -164,12 +165,13 @@ func addMount(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, k
 	case localPath:
 		lp = p
 	case gitRepo:
-		lp = localPath{path: p.basePath}
+		lp = localPath{p.basePath, p}
 	default:
 		return nil, fmt.Errorf(oldMountSyntaxError)
 	}
 
 	image.mounts = append(image.mounts, mount{lp, mountPoint})
+	image.repo = lp.repo
```

Comment:
what about the case where there's more than one repo?

Reply:
good call, fixed